src/index.js

```diff
@@ -3,7 +3,7 @@ import prettierFormatter from './formatters/prettier'
 import unstringSnapshotSerializer from './unstring-snapshot-serializer'
 
 // istanbul ignore else (it's not worth testing)
```

Comment:
Coming back to this it's kind of ironic that this caused a crash anyway :laughing: